release-notes/major9/09.25.0/valtimo-backend-libraries.md

```diff
@@ -39,7 +40,15 @@ The following bugs were fixed:
 
 The following breaking changes were introduced:
 
-* **Breaking change1**
+* **Case search field configuration**
+
+  Some breaking changes have been made regarding the case search field configurations that were introduced in
```

Comment:
No migration is necessary for this, right? Maybe that's worth mentioning here as well.